pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -25,13 +25,13 @@
 , stable ? true
```

Comment:
The commit message should be (IMO):

```
kicad: Rename `<feature>Support` arguments to `with<Feature>`

Also: Use assertions instead of silently ignoring arguments that don't cooperate
(occt+oce) / won't compile (aarch64 + oce)
```